addon-info.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I have no idea what this is. Is this like [contribute.json](https://www.contributejson.org/) but less info?